bedrock/security/templates/security/bug-bounty/faq.html

```diff
@@ -156,34 +149,31 @@ <h1 itemprop="name" class="title-shadow-box">Bug Bounty Program FAQ</h1>
             "fixed" solely as a byproduct of other unrelated changes.)</p>
         </dd>
 
-        <dt class="question" id="development-releases">Can I get the bug bounty
+        <dt id="development-releases">Can I get the bug bounty
           reward if I discover a bug in a pre-release version of Firefox, Firefox for
           Android, or Thunderbird?</dt>
-
-        <dd class="answer">
+        <dd>
           <p>Yes, as long as the bug otherwise meets the published bug bounty
-            program guidelines. Bugs found in Aurora and Beta releases of Firefox and
+            program guidelines. Bugs found in Developer Edition and Beta releases of Firefox and
             Firefox for Android, EarlyBird and Beta releases of Thunderbird are eligible as
             long the bug is reproducible in the latest nightly mozilla-central build and has
```

Comment:
I'm fine with changing this, but I'd rather it was done in a different bug since it's unrelated to the web bug bounty stuff.